lib/ImageResizer.js

```diff
@@ -17,6 +17,7 @@ ImageResizer.prototype.resizeRoute = function({originalRouteName, resizedRouteNa
   let width = this.options.width;
   let height = this.options.height;
   let allowEnlargement = this.options.allowEnlargement;
+  let disableRotation = this.options.disableRotation;
```

Comment:
Let's convert this to a positive statement, like `allowRotation`, to avoid confusion